src/dynamic_boundary_conditions/tide/sea_level_rise_data.py

```diff
@@ -27,14 +20,19 @@
 log = logging.getLogger(__name__)
 
 
-def download_slr_data_files_from_takiwa(slr_data_dir: pathlib.Path) -> None:
+def download_slr_data_files_from_takiwa(
+    slr_data_dir: pathlib.Path,
+    url: str = 'https://zenodo.org/records/11398538/export/json'
+) -> None:
     """
     Download regional sea level rise (SLR) data files from the NZ SeaRise Takiwa website.
 
     Parameters
     ----------
     slr_data_dir : pathlib.Path
         The directory where the downloaded sea level rise data files will be saved.
+    url: basestring
+        The url where sea level rise files need downloading.
```

Comment:
This url could be in the body of the function.

Alternatively, this function could take any zenodo url and download the files, in which case it should not have a default value and the naming should be modified.

Which do you think is better?

Reply:
I think maybe the first one for simple.